src/EventHandlers/Pool.ts

```diff
@@ -120,8 +137,12 @@ Pool.Swap.handlerWithLoader({
 
     if (liquidityPoolNew == undefined) return null;
 
-    const token0Instance = await context.Token.get(liquidityPoolNew.token0_id);
-    const token1Instance = await context.Token.get(liquidityPoolNew.token1_id);
+    const token0Instance = await context.Token.get(
+      liquidityPoolNew.token0_id.toLowerCase()
+    );
+    const token1Instance = await context.Token.get(
+      liquidityPoolNew.token1_id.toLowerCase()
```

Comment:
Again (beating a dead horse), but in this case the liquidity pool is a checksummed address, and the tokens are lowercase address :)